agent/05_agent_configure.sh

```diff
@@ -264,7 +371,20 @@ if [[ ! -z "${MIRROR_IMAGES}" ]]; then
 fi
 
 if [[ "${NUM_MASTERS}" > "1" ]]; then
-   set_api_and_ingress_vip
+  if [[ "${AGENT_PLATFORM_TYPE}" == "none" ]]; then
+    # for platform "none" both API and INGRESS point to the same
+    # load balancer IP address
+    get_vips
+    if [[ "$IP_STACK" = "v6" ]]; then
+      export load_balancer_ip=$(nth_ip $EXTERNAL_SUBNET_V6 1)
```

Comment:
`PROVISIONING_HOST_EXTERNAL_IP` could be used as well, will allow to simplify further

Reply:
Thanks, that does simplify it more.